lib/jekyll-seo-tag/json_ld.rb

```diff
@@ -18,62 +17,6 @@ module JSONLD
         :links          => "sameAs",
         :canonical_url  => "url",
       }.freeze
-
-      def json_ld
```

Comment:
`Drop` extended `JsonLD` (which is where I moved this method to). Theoretically, if someone extended the module into another class, this would be a breaking change. I'll can duplicate it here with a note that it should be removed at the next major release.